lesson/outline/outline-template.md

```diff
@@ -1,104 +1,323 @@
 
 
 ### What would you like to have in the workspace for this exercise? Share your plan below.
 
-_Check out the [content standards](http://curriculum-documentation.codecademy.com/Content-Standards/workspaces/) for guidance on writing narratives for exercises._
+- I.E. text editor and output terminal
 
-<hr>
 
-## Exercise # 3: _Insert exercise title here_
+## Exercise # 5: Testing Async Code in jest: Part 1
+
+### Which course outcomes will be covered by this exercise?
+
+1. Learners will be able to understand and solve false positives in jest using `done()`
+
+
+### Narrative Summary
+
+1. Begin by analyzing an api call that because of poor jest syntax, leads to a false positive, prematurely returning a test result.
+2. Explain why the test result returned as it did and how jest works with async code
+3. Re-write code to function properly
+Use done()
+
+
+### Checkpoints Summary
+
+Using a rest api call. 
+1. Rewrite test to use done(), using above example as reference
+
+ 
+#### What is the purpose of these checkpoints?
+
+- The learner will be able to write unit tests to analyze async code using done() 
+
+
+### What would you like to have in the workspace for this exercise? Share your plan below.
+
+- I.E. text editor and output terminal
+
+
+## Exercise # 6: Testing Async Code in jest: Part 2
 
 ### Which course outcomes will be covered by this exercise?
 
-1. Learners will be able to...
+1. Learners will be able to understand and solve false positives in jest using `.resolves`, `.rejects` and async/await. 
+
 
 ### Narrative Summary
 
-_Check out the [content standards](http://curriculum-documentation.codecademy.com/Content-Standards/narrative/) for guidance on writing narratives for exercises._
+1. Explain that there are two other ways to solve the false positive issue by introducing and explaining `.resolves`, `.rejects` and async/await
+2. Re-write code to function properly using `.resolves` and `.rejects`, explaining the structure and syntax
+3.  Re-write code to function properly using async/await, explaining the structure and syntax
+
+
+### Checkpoints Summary
+
+Using a rest api call. 
+1. Rewrite test to use `.resolves` and `.rejects`, using above example as reference
+2. Run test to verify
+3. Rewrite test to use async/await, using above example as reference
+4. Run test to verify
+
+ 
+#### What is the purpose of these checkpoints?
+
+- The learner will be able to write unit tests to analyze async code using `.resolves`, `.rejects` and async/await 
+
+
+### What would you like to have in the workspace for this exercise? Share your plan below.
+
+- I.E. text editor and output terminal
+
+
+## Exercise # 8: Mocking in jest: `jest.fn()`
+
+### Which course outcomes will be covered by this exercise?
+
+1. Learners will be able to understand testing psychology with regards to what aspects of our project are we specifically trying to testas well as mock an http request.
+
+
+
+### Narrative Summary
+
```

Comment:
Before diving straight into code, I think it would be important to talk about mocking in general. Why is it useful to make a fake function? You present mocking an API call as an example (which is great!) but keep in mind that the this is just one use case for mocking functions so be sure to communicate more generally the benefits that mocking functions provides.